workbench-core/swb-common-ui/src/components/Login.tsx

```diff
@@ -4,22 +4,21 @@
  */
 
 import Button from '@cloudscape-design/components/button';
+import * as React from 'react';
 import { useAuthentication } from '../context/AuthenticationContext';
-import styles from '../styles/Hero.module.scss';
```

Comment:
is this not supported anymore?

Reply:
There's only one style defined in it (ie class name `primaryButton`). I've applied this css on the dependent package (`solutions/swb-ui`) and just assigned the "primaryButton" class name to `swb-common-ui` component(s)

Reply:
I thought this was so we had autocomplete or build time detection but I may be wrong